api/pom.xml

```diff
@@ -103,7 +103,7 @@
             </resource>
         </resources>
         <plugins>
-            <!-- Sets minimal Maven version to 3.5.4 -->
+            <!-- Sets minimal Maven version to 3.6.0 -->
```

Comment:
Could be dropped or changed to `<!-- Sets minimal Maven version -->` to never need to be updated again or go out of sync.

Reply:
True, and I'm not such a fan of the message "You need Maven 3.6.0 or higher" either.